### DISCUSSION TITLE
The Grateful Thread

### DISCUSSION THREAD

**Discussion by mthlvt** [krahabb/meross_lan#58] [General]
Thank you so much for this integration! This is awesome!!

**Comment by krahabb** [comment]
Thank you !

**Comment by Joshndroid** [comment]
I came here just to start something like this.... good job OP.

@krahabb awesome integration, Have turned 4 tuya switches into now 8 meross switches with more to come based on your great work. Thanks again and supporting the community!

**Comment by barcus** [comment]
So much troubles with Merros Cloud iot on my side ! (using meross iot integration with default setup + rate limit enable)
My IP is blocked after 24h most of the time, i use ~20 devices !
Finaly discovered your integration, hope it will be beter :)
At least setup was really easy and everything seems to works out of the box, thank you for your job ^^

**Reply by Joshndroid** [reply]
I am at 8 devices at this time with more to come and working like a charm :)
